test/integration.sh

```diff
@@ -117,8 +117,7 @@ done
       run \
       --server \
       ./test/config \
-      --ignore '*.json' \
```

Comment:
Can this file not be removed now we're doing the integration tests in go?

Reply:
Yes, I thought I removed it together with `run-local.sh` :thinking: Must be still here by mistake. Removing.